test/test_data/GitleaksSampleData/sensitive_data.js

```diff
@@ -0,0 +1,3 @@
+// File containing sensitive AWS Access Key
+const secret = 'AKIAIOSFODNN8EXAMPLE';  // Example AWS access key
```

Comment:
This actually shouldn't be flagged because it's obviously false positive. The default Gitleaks `aws-access-token` rule explicitly ignores results that end in `EXAMPLE`.

https://github.com/gitleaks/gitleaks/blob/7f77987d91c590e021d9fa2b3c0e57c4a4376147/config/gitleaks.toml#L168-L171